Lib/idlelib/idle_test/test_colorizer.py

```diff
@@ -551,7 +551,7 @@ def test_long_multiline_string(self):
             ''')
         self._assert_highlighting(source, {'STRING': [('1.0', '5.4')]})
 
-    @run_in_tk_mainloop
+    @run_in_tk_mainloop(delay=50)
```

Comment:
What is this magic number? A number of seconds?

I suggest you to use test.support.SHORT_TIMEOUT or test.support.LONG_TIMEOUT: https://docs.python.org/dev/library/test.html#test.support.SHORT_TIMEOUT

Reply:
> What is this magic number? A number of seconds?

Milliseconds. It's explained in the decorator's doc-string.

> I suggest you to use test.support.SHORT_TIMEOUT or test.support.LONG_TIMEOUT

I wasn't aware of these, good to know! I don't think they're quite what's needed here though.

Reply:
> > What is this magic number? A number of seconds?
> 
> Milliseconds. It's explained in the decorator's doc-string.

Would you prefer that this was a `datetime.timedelta` instance?

Reply:
No, it's fine.